### DISCUSSION TITLE
NoClassDefFoundError DataLoaderDispatcherInstrumentationOptions

### DISCUSSION THREAD

**Discussion by thaparraj** [graphql-java-kickstart/graphql-java-servlet#577] [Q&A]
I am upgrading to graphql-java 22.0 from 21.0. I started getting the above Exception on startup. I did some investigation and found that the GraphQLQueryInvoker in com.graphql-java-kickstart:graphql-java-kickstart:15.1 .0 is refering to that class. And latest code in the repo removes that reference. Even after bumping the version to latest 15.2.0 of the lib, I am not getting the latest code from the repo! How should I resolve this Exception?

**Comment by samtawadhawan** [comment]
This class DataLoaderDispatcherInstrumentationOptions is only available till 21.5. So starting 22.0 it would be incompatible. Since the graphql-spring-boot-starter's latest version is 15.1.0 which uses 22.3 managed version of graphql-java, I would suggest to explicitly override version of graphql-java to 21.0 or 21.5.
